src/Badge/Badge.tsx

```diff
@@ -1,13 +1,22 @@
 import * as React from 'react';
 import { StyledBadge } from './Badge.styles';
 
-export interface Props {
+export interface BadgeProps {
+  /**
+   * The background colour
+   */
```

Comment:
Do we need these comments? We don't have them anywhere else.

Reply:
These are used to provide the additional description details in the docs table. I didn't fill them out for everything because the PR was already massive. I just wanted to provide an example of them for the future

![Screen Shot 2020-10-06 at 12 40 04](https://user-images.githubusercontent.com/20925005/95197154-12ce4780-07d1-11eb-8118-dadec3656c87.png)

Reply:
nice